packages/photon/src/generation/generateClient.ts

```diff
@@ -295,3 +305,143 @@ async function fileSize(name: string): Promise<number | null> {
+      `Extract${m.name}SelectCreateArgs`,
+      `Extract${m.name}IncludeCreateArgs`,
+
+      `${m.name}WhereInput`,
+      `${m.name}WhereUniqueInput`,
+      `${m.name}CreateInput`,
+      `${m.name}UpdateInput`,
+      `${m.name}UpdateManyMutationInput`,
+      `${m.name}OrderByInput`,
+     
+      `${m.name}CreateArgs`,
+      `${m.name}CreateArgsRequired`,
+      `${m.name}SelectCreateArgs`,
+      `${m.name}IncludeCreateArgs`,
+      `Extract${m.name}SelectCreateArgs`,
+      `Extract${m.name}IncludeCreateArgs`,
+      `${m.name}SelectCreateArgsOptional`,      
+      `${m.name}IncludeCreateArgsOptional`,      
+ 
+      `${m.name}UpsertArgs`,
+      `${m.name}UpsertArgsRequired`,
+      `${m.name}SelectUpsertArgs`,
+      `${m.name}IncludeUpsertArgs`,
+      `Extract${m.name}SelectUpsertArgs`,
+      `Extract${m.name}IncludeUpsertArgs`,
+      `${m.name}SelectUpsertArgsOptional`,      
+      `${m.name}IncludeUpsertArgsOptional`,      
+
+      `${m.name}UpdateArgs`,
+      `${m.name}UpdateArgsRequired`,
+      `${m.name}SelectUpdateArgs`,
+      `${m.name}IncludeUpdateArgs`,
+      `${m.name}UpdateManyArgs`,
+      `Extract${m.name}SelectUpdateArgs`,
+      `Extract${m.name}IncludeUpdateArgs`,
+      `${m.name}SelectUpdateArgsOptional`,      
+      `${m.name}IncludeUpdateArgsOptional`,      
+
+      `${m.name}DeleteArgs`,
+      `${m.name}DeleteArgsRequired`,
+      `${m.name}SelectDeleteArgs`,
+      `${m.name}IncludeDeleteArgs`,
+      `${m.name}DeleteManyArgs`,
+      `Extract${m.name}SelectDeleteArgs`,
+      `Extract${m.name}IncludeDeleteArgs`,
+      `${m.name}SelectDeleteArgsOptional`,      
+      `${m.name}IncludeDeleteArgsOptional`,
+
+      `FindOne${m.name}Args`,
+      `FindOne${m.name}ArgsRequired`,
+      `FindOne${m.name}SelectArgs`,
+      `FindOne${m.name}IncludeArgs`,
+      `FindOne${m.name}SelectArgsOptional`,
+      `FindOne${m.name}IncludeArgsOptional`,
+      `ExtractFindOne${m.name}SelectArgs`,
+      `ExtractFindOne${m.name}IncludeArgs`,
+     
+      `FindMany${m.name}Args`,
+      `FindMany${m.name}ArgsRequired`,
+      `FindMany${m.name}IncludeArgs`,
+      `FindMany${m.name}SelectArgs`,
+      `FindMany${m.name}SelectArgsOptional`,
+      `FindMany${m.name}IncludeArgsOptional`,
+      `ExtractFindMany${m.name}SelectArgs`,
+      `ExtractFindMany${m.name}IncludeArgs`,
+    ])
+  }
+
+  if (prismaClientDmmf.datamodel.enums) {
+    for (const it of prismaClientDmmf.datamodel.enums) {
+      if (
+        denylists.models.includes(it.name) ||
+        denylists.fields.includes(it.name) ||
+        denylists.dynamic.includes(it.name)) {
+        errorArray.push(Error(`"enum ${it.name}"`))
+      }
+    }
+  }
+  
+  if (prismaClientDmmf.datamodel.enums) {
+    for (const it of prismaClientDmmf.datamodel.models) {
+      if (
+        denylists.models.includes(it.name) ||
+        denylists.fields.includes(it.name)) {
+        errorArray.push(Error(`"model ${it.name}"`))
```

Comment:
Do I understand correctly that this is the list of dynamic strings that are disallowed in other models (or enum names) because they are generated based on an existing model? (If we have a model `foo` then `fooSelect` is denied)

Reply:
The dynamic list doesn't apply to model names, only to enum names.

Recent example: a user created a model `Maker` and an enum `MakerArgs`. This caused an error when trying to run the code, it will now error during the code generation.

Reply:
Oh interesting.